src/main/scala/com/microsoft/partnercatalyst/fortis/spark/Settings.scala

```diff
@@ -0,0 +1,10 @@
+package com.microsoft.partnercatalyst.fortis.spark
+
+trait Settings {
```

Comment:
Here the trait is a good choice since we'll have multiple implementations. Right now we have an `EnvironmentVariablesSettings` and in the future we'll have a `CasandraSettings`.